packages/swing-store-simple/test/test-state.js

```diff
@@ -72,3 +72,100 @@ test('rejectLMDB', t => {
   fs.writeFileSync(path.resolve(notSimpleDir, 'lock.mdb'), 'lock stuff\n');
   t.is(isSwingStore(notSimpleDir), false);
 });
+
+test('streamStore read/write', t => {
+  const { streamStore, commit, close } = initSwingStore();
+
+  let s1pos;
+  const writer1 = streamStore.openWriteStream('st1');
+  s1pos = writer1('first', s1pos);
+  s1pos = writer1('second', s1pos);
+  const s1posAlt = { ...s1pos };
+  const writer2 = streamStore.openWriteStream('st2');
+  s1pos = writer1('third', s1pos);
+  let s2pos = { itemCount: 0 };
+  s2pos = writer2('oneth', s2pos);
+  s1pos = writer1('fourth', s1pos);
+  s2pos = writer2('twoth', s2pos);
+  const s2posAlt = { ...s2pos };
+  s2pos = writer2('threeth', s2pos);
+  s2pos = writer2('fourst', s2pos);
+  streamStore.closeStream('st1');
+  streamStore.closeStream('st2');
+  const reader1 = streamStore.openReadStream('st1', s1pos);
+  const reads1 = [];
+  for (const item of reader1) {
```

Comment:
would `const reads1 = Array.from(reader1)` be equivalent? not that it should be changed, just checking my understanding of iterators

Reply:
I think that's right.  I wrote it as a loop in something between force of habit and thinking about what might happen if iteration was not complete (though the latter case is handled by a different test).